pkg/zson/docs/spec.md

```diff
@@ -201,24 +197,24 @@ by a value encoding.  Here is a pseudo-grammar for value encodings:
         | <list> <elem> ;
 <terminal> := <char>*
 <char> := [^\n\\;[]] | <esc-sequence>
-<esc-sequence> := <javascript character escaping rules>
+<esc-sequence> := <JavaScript character escaping rules>
 ```
-A value is encoded a string of utf-8 characters terminated
+A value is encoded as a string of UTF-8 characters terminated
 by a semicolon (which must be escaped if it appears in the value) where
 composite values are contained in brackets as one or more such
-semi-colon terminated strings.  Any escaped characters shall be processed
+semicolon terminated strings.  Any escaped characters shall be processed
 and interpreted as their escaped value.
 
 Container values are encoded as
 * an open bracket,
 * zero or more encoded values, and
 * a close bracket.
 
-Any value can be specified as "unset" with the ascii character '-'.
+Any value can be specified as "unset" with the ascii character `-`.
 This is typically used to represent columns of records where not all
-columns have been set in a given record value though any type can be
+columns have been set in a given record value, though any type can be
 validly unset.  A value that is not to be interpreted as "unset"
-but is the single-character string "-", must be escaped (e.g., "\-").
+but is the single-character string `-`, must be escaped (e.g., `\-`).
```

Comment:
I kept the `\-` as the escaped representation since that's what was here before, but is this the right way to represent it? The "Character Escape Rules" above doesn't explicitly call this one out, unless it's covered by the "JavaScript character escaping rules". But this is slightly confusing to me because the JavaScript references I find online don't appear to treat `-` as a special character that needs escaping. If we were in the legacy zone I'd use the `\x` hex escaping, but we're in the regular zone.